Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -243,6 +267,14 @@ void MarkupAccumulator::serializeNodesWithNamespaces(Node& targetNode, Serialize
     } else
         namespaceStack.constructAndAppend();
 
+    auto getFirstChild = [&](auto& current) -> Node* {
```

Comment:
"get" prefix should only be used on a function with an out argument.
https://webkit.org/code-style-guidelines/#names-out-argument

Reply:
Maybe `computeFirstChild` or simply `firstChild`?